ts/IterableInAppMessage.ts

```diff
@@ -84,6 +84,21 @@ class IterableInAppMessage {
     this.priorityLevel = priorityLevel
   }
 
+  static fromInApp(inAppMessage: IterableInAppMessage) {
+    return new IterableInAppMessage(
```

Comment:
Not sure but looks strange that the class can accept the object of same type and return a new class out of it.. 😐 might need Patrick or Martys opinion to see if it is the best way.

Reply:
There's no way to create more constructors, and for some reason, this is necessary to avoid errors within TypeScript. Will ask a TS expert.

Reply:
After talking with Marty and Patrick about this, the OVERALL way I'm doing this is probably the best way to handle it due to it being a weird API, but a suggested change would be to have `IterableInAppMessage.ts` take a `ViewToken` rather than an `IterableInAppMessage` here.

I'll do this as a separate ticket since it's only relevant to this specific feature and functionally doesn't change anything.

Reply:
sounds good 👍